kernel/src/main/java/org/kframework/backend/kore/ModuleToKORE.java

```diff
@@ -976,7 +976,7 @@ private Att addKoreAttributes(Production prod, SetMultimap<KLabel, Rule> functio
         if (isMacro) {
             att = att.add("macro");
         }
-        return att;
+        return att.remove(Constants.ORIGINAL_PRD, Production.class);
```

Comment:
is this change related to Production_ID?

Reply:
No, it's related to the upcoming changes with polymorphic productions. We are removing this attribute because it doesn't belong in the generated kore and it was messing with my ability to compare the kore generated before and after my changes. I should be able to delete this line now that I'm done testing if you'd like though?

Reply:
can you add a todo here to make sure that we don't forget it?

Reply:
It's ok to remove this attribute since it is never used in kore generation. I would add a comment here pointing out this attribute is useless in the kore generation if possible.